zerver/actions/realm_settings.py

```diff
@@ -791,7 +791,10 @@ def do_change_realm_plan_type(
         "op": "update",
         "property": "plan_type",
         "value": plan_type,
-        "extra_data": {"upload_quota": realm.upload_quota_bytes()},
+        "extra_data": {
+            "upload_quota": realm.upload_quota_bytes(),
+            "max_file_upload_size_mib": realm.get_max_file_upload_size_mebibytes(),
```

Comment:
Do we want this to be attached to this, or would it be better to just have a second event with a `property` of `max_file_upload_size_mib`? I feel like mobile clients probably don't want a super custom `plan_type` event.; my guess is this already somewhat weird event shape only exists because the data is only used in web...

Reply:
https://chat.zulip.org/#narrow/stream/378-api-design/topic/max.20file.20size.20upload.20limit/near/1954928